resource/transform.go

```diff
@@ -391,7 +391,11 @@ func (r *transformedResource) transform(setContent bool) (err error) {
 				// If a prepared bundle for this transformation chain is available, use that.
 				f := r.tryTransformedFileCache(key)
 				if f == nil {
-					return fmt.Errorf("%s: failed to transform %q (%s): %s", strings.ToUpper(tr.transformation.Key().name), tctx.InPath, tctx.InMediaType.Type(), err)
+					errMsg := err.Error()
+					if tr.transformation.Key().name == "postcss" {
```

Comment:
You should also check that this is a FeatureNotAvailableErr -- you may get other errors where this error message may confuse more than it helps

Reply:
Thank you for reviewing!

> You should also check that this is a FeatureNotAvailableErr -- you may get other errors where this error message may confuse more than it helps

If I understand correctly, this check for FeatureNotAvailableErr is already done a few lines back.  My changes are with an `if err == errors.FeatureNotAvailableErr { }` block which already exists in the code, see https://github.com/gohugoio/hugo/blob/94d6d6780fac78e9ed5ed58ecdb9821ad8f0f27c/resource/transform.go#L387-L405

Seeing that `err` is not redefined within this `if` block, I suppose I do not need to check it again?  :-)